pulsar-broker-auth-sasl/src/main/java/org/apache/pulsar/broker/authentication/PulsarSaslServer.java

```diff
@@ -0,0 +1,198 @@
+/*
+ Licensed to Diennea S.r.l. under one
```

Comment:
This will fail the License check. Either we add an exclusion or we ask the copyirght holder to donate this code to ASF.

Reply:
@merlimat, @eolivelli  right, it is copy from bookkeeper before this change. and It is there in BookKeeper.

Reply:
I will send a patch for Bookkeeper